.travis.yml

```diff
@@ -1,24 +1,19 @@
 language: python
+python:
+  - "3.5"
 install:
   - pip install coveralls tox
 script:
   - tox
 env:  # generate list with: $ tox -l | xargs -I ITEM echo "  - TOXENV="ITEM
-  - TOXENV=py26-django14
-  - TOXENV=py26-django15
-  - TOXENV=py26-django16
-  - TOXENV=py27-django14
-  - TOXENV=py27-django15
-  - TOXENV=py27-django16
   - TOXENV=py27-django17
-  - TOXENV=py27-django18
-  - TOXENV=py32-django15
-  - TOXENV=py32-django16
   - TOXENV=py32-django17
-  - TOXENV=py32-django18
-  - TOXENV=py33-django15
-  - TOXENV=py33-django16
   - TOXENV=py33-django17
-  - TOXENV=py33-django18
   - TOXENV=py34-django17
+  - TOXENV=py33-django18
```

Comment:
This is actually auto-generated code, so I don't think that really matters. I'd love to see this disappear in favour of some programatical approach that does not require manually updating this section.

khal (totally unrelated) deals nicely with this:

https://github.com/geier/khal/blob/master/.travis.yml
https://github.com/geier/khal/blob/master/.travis.run.sh